modin/core/dataframe/algebra/map.py

```diff
@@ -43,9 +43,7 @@ def register(cls, function, *call_args, **call_kwds):
 
         def caller(query_compiler, *args, **kwargs):
             """Execute Map function against passed query compiler."""
-            shape_hint = call_kwds.pop("shape_hint", None) or kwargs.pop(
-                "shape_hint", None
-            )
+            shape_hint = call_kwds.pop("shape_hint", None) or query_compiler._shape_hint
```

Comment:
So there are no map operations in Series that return DataFrames?

Reply:
map operator preserves the shape of an object.